Documentation/troubleshooting.md

```diff
@@ -198,3 +198,64 @@ kubectl get pods --all-namespaces | grep 'prom.*operator'
 ```
 
 Check the logs of the matching pods to see if they manage the same resource.
+
+### Configuring Prometheus/PrometheusAgent for Mimir and Grafana Cloud
+
+Mimir and Grafana Cloud can receive samples via Prometheus remote-write and are able to [deduplicate samples](https://grafana.com/docs/mimir/latest/configure/configure-high-availability-deduplication/) received from HA pairs of Prometheus/PrometheusAgent instances provided that you configure proper labels.
+
+By default, the deduplication labels are:
+* `cluster`, it identifies the HA pair and should have the same value for both instances.
+* `__replica__`, it should have a different value for each instance.
+
+The Prometheus operator already configures the `prometheus_replica` external label with the same semantic than `__replica__`. The label name can be changed to `__replica__` by setting the `.spec.replicaExternalLabelName` field. When running a self-managed Mimir, it's also possible to configure different deduplication labels on Mimir side (check the Mimir documentation).
+
+When it's not possible to change the Prometheus replica external label, a simple solution is to leverage `writeRelabelConfigs`. Here is a full example:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: example
+spec:
+  podMonitorSelector: {}
+  ruleSelector: {}
+  serviceMonitorSelector: {}
```

Comment:
Is this necessary for this example, as we are only concerned with showing how to leverage `writeRelabelConfigs`?

Reply:
It's to avoid the "but I tried the example from the docs and it didn't work!".

Reply:
Yeah, agreed.